src/main/java/com/crud_read_and_create/controller/GameController.java

```diff
@@ -84,30 +87,28 @@ public String search(GameForm gameForm, Model model) {
 			} else {
 				model.addAttribute("mojiretsu", "数字を入力して下さい。");
 			}
-
 		}
-
 		return "search/db";
 	}
 
 	@PostMapping("/create")
-	public String create(@ModelAttribute @Validated GameForm gameForm, BindingResult bindingResult, Model model) {
+	public String create(@ModelAttribute @Validated GameForm gameForm, BindingResult bindingResult,
+			GamePlatformForm gamePlatformForm, Model model) {
 
 		if (bindingResult.hasErrors()) {
 			model.addAttribute("createFailed", "登録に失敗しました。");
-			return "/create";
-
 		} else {
-			gameService.create(gameForm);
+			gameService.createGame(gameForm, gamePlatformForm);
 			model.addAttribute("createSuccess", "登録に成功しました。");
 		}
-
+		List<Platform> platformList = gameService.getPlatform();
+		model.addAttribute("platformList", platformList);
 		return "/create";
```

Comment:
PRGパターンにしましょう。
```suggestion
		return "redirect:/create";
```

PRGパターンとその利点について調べてみてください。
キーワードは
`PRGパターン`
`PRGパターン spring`

それから、ただ"redirect:/create"に変えただけだと、「登録に成功しました。」といったメッセージが表示されなくなるはずです。
やり方は複数ありますが`RedirectAttributes`を使うのがいいかと思います。

PRG springなどで調べると自然とRedirectAttributesでパラメータを渡して云々という記事などが見つかると思います。

Reply:
https://github.com/renangton/task5_crud_read_and_create/pull/15/commits/9bcd3a8bbd6fdd53dafc6e2a0f0de8359bc805a4
対応しました。
登録成功時はPRGパターンを適用し、登録失敗時はPRGパターンを適用していません。